proof-producer/libs/assigner/include/nil/proof-generator/assigner/rw.hpp

```diff
@@ -38,6 +38,12 @@ namespace nil {
             for (const auto& storage_op : rw_operations->storage_ops) {
                 input.push_back(storage_op);
             }
+
+            using nil::blueprint::bbf::rw_operation;
```

Comment:
Let's move sort to `deserialize_rw_traces_from_file`, sorted rw operation used by another circuits, like zkevm

Reply:
zkevm and other circuits do not require operations to be sorted, and I am not sure if it breaks something or not
Will check

Reply:
Actually nothing broke, got rid of struct RWOperations and now do the sort inside trace parser